apitools/base/py/credentials_lib.py

```diff
@@ -384,6 +387,21 @@ def _refresh(self, _):
     self.access_token = token
 
 
+def _GetRunFlowFlags():
+  if FLAGS is None:
+    parser = argparse.ArgumentParser(parents=[tools.argparser])
+    # We pass [] to just use the defaults from tools.argparser instead of
+    # using command line flags.
+    return parser.parse_args([])
```

Comment:
@craigcitro Note that this disables using the arguments (i.e. `sys.argv`). Do you want to allow command line flags for both `argparse` and `gflags`? It's a simple change to `parser.parse_args()` (no empty list passed) if we want to support `argparse` from the command line as well.